.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+ - "10.14.1"
+before_script:
+  - npm install
+script: 
+  - npm run test
```

Comment:
Travis should still work as expected even if there's no script directive in this `.travis.yml`, so you could take that out.

By default, it is designed to find and run `npm test`

Reply:
Ok, I never knew about that. Thanks for the feedback. I will be taken it out asap.

Reply:
Oh, that's fine @XROLE I know you wouldn't have put it there if you knew.

Reply:
### *Please,* I need clarification on these two issues. 
1. Having removed `npm install` and `npm run test`, the build still passed on Travis as expected but on Github it kept on telling me that 1 (one) check is still pending.
2. The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait untill the badge color changes to success (green)?

Reply:
> I'd need you to add the travis badge to the `README.md` @XROLE

Ok. I am adding it right away.

Reply:
> #### Please, I need clarification on this;
> * The build passed on Travis CI successfully but the badge has not changed color to show that the test is passing. Do I still add the badge that way or do I wait until the badge color changes to success (green)?

The reason the badge is not changing color is that we need the develop version of the CI. However, at the moment, travis is not on develop yet, so you'd most likely only see an `unknown` status. As long as the travis build passes, when we merge to develop, the badge will be green.

Reply:
Ok. Thanks so much for that.